[2024-01-11T18:16:09.811+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2024-01-11T18:16:02+00:00 [queued]>
[2024-01-11T18:16:09.826+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2024-01-11T18:16:02+00:00 [queued]>
[2024-01-11T18:16:09.826+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-11T18:16:09.839+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): runme_2> on 2024-01-11 18:16:02+00:00
[2024-01-11T18:16:09.847+0000] {standard_task_runner.py:60} INFO - Started process 186 to run task
[2024-01-11T18:16:09.868+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'manual__2024-01-11T18:16:02+00:00', '--job-id', '11', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp6szh4vlg']
[2024-01-11T18:16:09.898+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask runme_2
[2024-01-11T18:16:10.003+0000] {task_command.py:423} INFO - Running <TaskInstance: example_bash_operator.runme_2 manual__2024-01-11T18:16:02+00:00 [running]> on host ca44f6985e16
[2024-01-11T18:16:10.165+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_2' AIRFLOW_CTX_EXECUTION_DATE='2024-01-11T18:16:02+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-11T18:16:02+00:00'
[2024-01-11T18:16:10.167+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-11T18:16:10.167+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "example_bash_operator__runme_2__20240111" && sleep 1']
[2024-01-11T18:16:10.187+0000] {subprocess.py:86} INFO - Output:
[2024-01-11T18:16:10.197+0000] {subprocess.py:93} INFO - example_bash_operator__runme_2__20240111
[2024-01-11T18:16:11.201+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-11T18:16:11.223+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20240111T181602, start_date=20240111T181609, end_date=20240111T181611
[2024-01-11T18:16:11.272+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-11T18:16:11.288+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
